### DISCUSSION TITLE
Help with JVM args.

### DISCUSSION THREAD

**Discussion by DONOcoding** [TheUsefulLists/UsefulMods#207] [Q&A] [answered]
Hello, im making a modpack for a couple of friends and i want to add jvm arguments. but in 1.18.2 forge has changed. so now we do not have the "forge-1.18.2-38.0.2.jar" to launch the server, now we have run.sh. i wanted to learn how to put args in this new system plz cause i found this list and i would like to use it.

**Comment by NordicGamerFE** [comment, marked as answer]
you can find the java run file in libraries/net/minecraftforge/forge/
